content/bn/docs/concepts/cluster-administration/_index.md

```diff
@@ -60,21 +61,21 @@ card:
 * [কুবারনেটিস কন্টেইনার এনভায়রনমেন্ট](/bn/docs/concepts/containers/container-environment/) একটি কুবারনেটিস
   নোডে Kubelet পরিচালিত কন্টেইনারগুলির পরিবেশ বর্ণনা করে।
   
-* [Kubernetes API-তে অ্যাক্সেস নিয়ন্ত্রণ](/bn/docs/concepts/security/controlling-access) বর্ণনা করে
-  কিভাবে কুবারনেটিস তার নিজস্ব API এর জন্য অ্যাক্সেস নিয়ন্ত্রণ প্রয়োগ করে।
+* [Kubernetes API-তে অ্যাক্সেস কন্ট্রোল](/bn/docs/concepts/security/controlling-access) বর্ণনা করে
+  কিভাবে কুবারনেটিস তার নিজস্ব API এর জন্য অ্যাক্সেস কন্ট্রোল প্রয়োগ করে।
 
-* [যাচাইকরণ](/bn/docs/reference/access-authn-authz/authentication/) বিভিন্ন যাচাইকরণ বিকল্প সহ,
-  কুবারনেটিসে যাচাইকরণের ব্যাখ্যা দেয়।
+* [অথেন্টিকেশন](/bn/docs/reference/access-authn-authz/authentication/) বিভিন্ন অথেন্টিকেশন বিকল্প সহ,
+  কুবারনেটিসে অথেন্টিকেশনের ব্যাখ্যা দেয়।
 
-* [অথোরাইজেশন](/bn/docs/reference/access-authn-authz/authorization/) যাচাইকরণ থেকে আলাদা,
+* [অথোরাইজেশন](/bn/docs/reference/access-authn-authz/authorization/) অথেন্টিকেশন থেকে আলাদা,
   এবং HTTP কলগুলি কীভাবে পরিচালনা করা হয় তা নিয়ন্ত্রণ করে।
 
-* [ভর্তি নিয়ন্ত্রকদের ব্যবহার](/bn/docs/reference/access-authn-authz/admission-controllers/)
-  প্লাগ-ইনগুলি ব্যাখ্যা করে যা যাচাইকরণ এবং অনুমোদনের পরে কুবারনেটস API সার্ভারে 
+* [অ্যাডমিশন কন্ট্রোলের ব্যবহার](/bn/docs/reference/access-authn-authz/admission-controllers/)
+  ব্যাখ্যা করে প্লাগ-ইনগুলি প্লাগ-ইনগুলি অথেন্টিকেশন এবং অথোরাইজেশনের পরে কুবারনেটস API সার্ভারে

```

Comment:
ব্যাখ্যা করে প্লাগ-ইনগুলি অথেন্টিকেশন এবং অথোরাইজেশনের পরে কুবারনেটস API সার্ভারে

Reply:
sorry that's my mistake , i write প্লাগ-ইনগুলি twice